images/virtualization-artifact/pkg/controller/vm/internal/block_device.go

```diff
@@ -299,9 +300,16 @@ func (h *BlockDeviceHandler) countReadyBlockDevices(vm *virtv2.VirtualMachine, s
 				canStartKVVM = false
 				continue
 			}
-
-			if vd.Status.Phase == virtv2.DiskReady {
-				ready++
+			readyCondition, _ := service.GetCondition(vdcondition.ReadyType, vd.Status.Conditions)
+			if readyCondition.Status == metav1.ConditionTrue {
+				inUseByVICondition, _ := service.GetCondition(vdcondition.InUseType, vd.Status.Conditions)
+				if inUseByVICondition.Status != metav1.ConditionTrue || inUseByVICondition.Reason == vdcondition.InUseByVirtualMachine {
+					ready++
+				} else {
+					msg := fmt.Sprintf("The virtual disk %s is being used to create a image.", vd.Name)
```

Comment:
It has to start with a lowercase letter, and there should be no period at the end